python/step_4.html

```diff
@@ -0,0 +1,73 @@
+<!--TODO this is way too long, but I don't know how to break it down, there is some theory on Rest and then some necessary info on pokerwars-->
+
+<div class="instructions">
+    <h1>RESTful Endpoints</h1>
+    <p>
+        Hey good to see you again. In this step we are going to have fun learning some very interesting concept about
+        API design.
+    </p>
+    <h2>The theory</h2>
+    <p>
+        So API are the connections between your software and the outside world, that can be another service in your
```

Comment:
words like "service" and "domain" sounds difficult to me, can we avoid them? Or we need to explain what they are before (I'd prefer to avoid).

Reply:
ok